package.json

```diff
@@ -26,16 +27,19 @@
   "homepage": "https://github.com/webcompat/webcompat-reporter-extensions#readme",
   "devDependencies": {
     "copy-webpack-plugin": "^4.3.1",
+    "crx": "^3.2.1",
     "eslint": "4.16.0",
     "eslint-config-prettier": "^1.7.0",
     "eslint-plugin-prettier": "^2.0.1",
     "file-exists": "^5.0.1",
+    "foo": "0.0.7",
```

Comment:
Oops!

This is totally my fault, because I gave you the following command: `(the typical way to add something as a devDependency to a package.json is via npm install --save-dev foo)`

(sometimes programmers use `foo` or `bar` as a placeholder or nonsense variable or name).

Can you remove this `foo` package? Then we're good to merge!

Reply:
Ohhh good to know.
It’s interesting that there is a package called ‘foo’, no wonder I didn’t get an error. 😅

Reply:
I will fix it asap. :)